templates/_components/free_text.html.twig

```diff
@@ -1,18 +1,19 @@
-<div class="govuk-width-container">
-    <div class="govuk-border-free-text"
+    <div class="govuk-width-container">
     <div class="govuk-grid-row">

```

Comment:
Can you please indent this file based on the depth of the HTML elements?

For example, at the moment you have this:

```
    <div class="govuk-width-container">
    <div class="govuk-grid-row">
```

This should be:

```
    <div class="govuk-width-container">
        <div class="govuk-grid-row">
```

Indenting the code like this will make it easier to read and understand.